webrender_api/src/font.rs

```diff
@@ -268,14 +268,16 @@ impl Default for FontInstanceOptions {
 #[repr(C)]
 #[derive(Clone, Copy, Debug, Deserialize, Hash, Eq, PartialEq, PartialOrd, Ord, Serialize)]
 pub struct FontInstancePlatformOptions {
-    pub unused: u32,
+    pub gamma: u16, // percent
+    pub contrast: u16, // percent
 }
 
 #[cfg(target_os = "windows")]
 impl Default for FontInstancePlatformOptions {
     fn default() -> FontInstancePlatformOptions {
         FontInstancePlatformOptions {
-            unused: 0,
+            gamma: 220,
```

Comment:
The previous default constant used was 230 - is 220 correct? Also, could we add a comment here describing why this is the default?